setup.py

```diff
@@ -18,7 +18,7 @@
 NAME = "enplot"
 MAJOR = 1
 MINOR = 0
-MICRO = 1
+MICRO = 2
```

Comment:
Let's make it 1.1.0 instead of 1.0.2 since this PR adds new features (log transforms)